lib/workers/repository/onboarding/branch/config.ts

```diff
@@ -12,7 +14,7 @@ import { EditorConfig, JSONWriter } from '../../../../util/json-writer';
 async function getOnboardingConfig(
   config: RenovateConfig,
 ): Promise<RenovateSharedConfig | undefined> {
-  let onboardingConfig = clone(config.onboardingConfig);
+  let onboardingConfig = clone(config.onboardingConfig) ?? {};
```

Comment:
? why the fallback 🤔 onboarding config should never null or undefined here.

Reply:
According to types it could be